Source/WebKit/UIProcess/API/C/WKOpenPanelResultListener.cpp

```diff
@@ -59,16 +59,21 @@ static Vector<String> filePathsFromFileURLs(const API::Array& fileURLs)
 #if PLATFORM(IOS_FAMILY)
 void WKOpenPanelResultListenerChooseMediaFiles(WKOpenPanelResultListenerRef listenerRef, WKArrayRef fileURLsRef, WKStringRef displayString, WKDataRef iconImageDataRef)
 {
-    toImpl(listenerRef)->chooseFiles(filePathsFromFileURLs(*toImpl(fileURLsRef)), toImpl(displayString)->string(), toImpl(iconImageDataRef));
+    if (auto protectedFileURLs = toProtectedImpl(fileURLsRef)) {
```

Comment:
Why the new null check? Also, we don't use `protected` functions to initialize local variables nor do we hide smart pointers behind `auto`.